tbx/project_styleguide/templates/patterns/organisms/footer/footer.html

```diff
@@ -11,6 +10,7 @@
             {% wagtailcache 600 "footerlinks" current_site.pk is_pattern_library %}
                 {% footerlinks %}
             {% endwagtailcache %}
+            <a href="#" class="footer__newsletter-link"><span class="footer__newsletter-link-text"><span class="footer__newsletter-link-title">Get news and insights </span>Sign up for our newsletter</span>{% include "patterns/atoms/icons/icon.html" with name="arrow-wide" classname="footer__arrow" %}</a>
```

Comment:
Thought: I noticed that we haven't included an underline on this link and looking more into the design it's been removed from a few other places e.g. links in related content:

Design:

<img width="1756" alt="image" src="https://github.com/user-attachments/assets/c8b8419d-0942-4cd6-9017-a169ae4b3fe8" />

Current site:

<img width="1503" alt="image" src="https://github.com/user-attachments/assets/d409cc2f-49c1-4d53-8305-6c4bed6ace65" />

The current site uses an underline to signify a link so it might be worth speaking to design to make sure we're consistent on link styles so we don't have a mix of some underlined and some not.

Reply:
another excellent find - we're updating all link styles to include underlines. For this particular link, we're reusing underlines from the footernav link styles